### PR TITLE
update sortable table story to fix CustomHeader sorting

### PR DESCRIPTION
**Pull request by aojin**
The associated fix to Storybook SortableTable for CustomHeader. We were not passing down the onClick handler.

### CODE REVIEW

File: src/tables/table-column.js [hunk 1]
```diff
@@ -9,8 +9,16 @@ import { columnPropTypes } from './helpers'
  * @param {String} [label] - The text that will be displayed in the column header. Defaults to `name`.
  * @param {Function} [sortFunc] - The function that will be used to sort the table data when the column is selected
  * @param {Function} [component] - A custom cell component for the column. Will be passed the `key`, `name`, `value` and `data` for the row.
- * @param {Function} [headerComponent] - A custom header component for the column. Will be passed the configuration of the column, as well as the current `sortPath` / `ascending` and an `onClick` handler.
- * @param {Function} [onClick] - A function that will be called `onClick` on every cell in the column
+ * @param {Function} [headerComponent] - A custom header component for the column. Will be passed the configuration of the column, as well as the current `sortPath` / `ascending` and an `onClick` handler. onClick must be appended to allow for sorting functionality.
```
Comment (chawes13) [inline src/tables/table-column.js:12]: To be consistent, can you wrap "onClick" in single backticks?

File: src/tables/table-column.js [hunk 2]
```diff
@@ -9,8 +9,16 @@ import { columnPropTypes } from './helpers'
...
+ * @example
+ * 
+ * function CustomHeader({ column: { name }, onClick }) {
+ *   return (
+ *     <th onClick={onClick}>{name.toUpperCase() + '!'}</th>
+ *   )
+ * }
+ *
```
Comment (chawes13) [inline src/tables/table-column.js:20]: I think we should have all examples _after_ the params have all been declared

**Review by chawes13 — CHANGES_REQUESTED** [review]
Few minor comments in-line. Can you bump the version and remove all the .DS_Store documents? I'll create a separate issue to make sure those are added to `.gitignore` in the future

**Comment by chawes13** [comment]
@aojin Looks good, just needs a version bump! This change can be made in `package.json`.

**Comment by chawes13** [comment]
@aojin Code looks good -- needs a version bump to pass versionator's check

**Review by chawes13 — CHANGES_REQUESTED** [review]
Version in `package.json`

**Comment by aojin** [comment]
Sorry for the delay! Version bumped to 5.4.2!

**Review by chawes13 — APPROVED** [review]
🚀

**Pull request merged**